src/main/java/it/gov/pagopa/pu/sil/connector/actualization/NativeActualizationService.java

```diff
@@ -0,0 +1,10 @@
+package it.gov.pagopa.pu.sil.connector.actualization;
+
+import it.gov.pagopa.actualization.dto.generated.Payment;
+import it.gov.pagopa.actualization.dto.generated.UpdatedPayment;
+import it.gov.pagopa.pu.auth.dto.generated.UserInfo;
+import it.gov.pagopa.pu.organization.dto.generated.OrgSilServiceDTO;
+
+public interface NativeActualizationService {
```

Comment:
let's not use `Native` suffix, it's implicit
```suggestion
public interface ActualizationService {
```

Reply:
done